system_setup/server/controllers/shutdown.py

```diff
@@ -50,18 +50,24 @@ async def _wait_install(self):
     @with_context(description='mode={self.mode.name}')
     def shutdown(self, context):
         self.shuttingdown_event.set()
-        launcher_status = "complete"
+        comments = ["# This file was auto generated by subiquity server.",
```

Comment:
Nitpick: random users may not understand the server portion - or subiquity in general.  Maybe just call it system-setup?

Reply:
That's fine. I'll go with your suggestion.